binder/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM uwhackweeks/icesat2:2020.06.15
```

Comment:
With the most recent updates to `icepyx`, if we try to build against this Hackweek Docker image the examples will fail, since the `icepyx` example notebooks have been updated to use `Query` objects instead of `Icesat2Data` objects.